### ISSUE TITLE
[IDEA] - proxy detect

### ISSUE THREAD

**Issue by Valdisa4** [xDefcon/sinusbot-scripts#15]
heyy it would be nice if you would create an option that if a proxy is detected on a person you can add him to a specific group. like for example server mute. and make an option "ignore groups" and groups that are in that section are ignored of being banned / checked if they are using proxy.

why did this idea came in my mind ? its because in my server every member is checked if they are allowed to send messages and stuff like that. if they are aproved they get the group "Normal" and if they are using proxy, i dont really care, they dont harm me in anyway, but people with "guest" group that join and spam music, kinda annoy me. and if a proxy is detected on a member with "Guest" group - the script just adds him to a "anti-join" group, something like that, and then i can just go and check them if they are not a spam bot or some random annoying person.

did you get what i meant ? or should i try explaining that in a different way :D ?

basically , People with "GUEST" group get checked if they are using proxy , if they are then they are added to a different server group
people with "normal" group dont get check/banned/kicked of what ever , even tho they are using proxy.
